dynamax/slds/mixture_kalman_filter.py

```diff
@@ -0,0 +1,161 @@
+# Mixture Kalman Filter library. Also known as the
+# Rao-Blackwell Particle Filter.
+
+# Author: Gerardo Durán-Martín (@gerdm)
+
+import jax
+import jax.numpy as jnp
+from jax import random
+from jax.scipy.special import logit
+from dataclasses import dataclass
+
+
+@dataclass
+class RBPFParamsDiscrete:
+    """
+    Rao-Blackwell Particle Filtering (RBPF) parameters for
+    a system with discrete latent-space.
+    We assume that the system evolves as
+        z_next = A * z_old + B(u_old) + noise1_next
```

Comment:
So why is the bias term the only one that depends on the discrete latent state u_old?

Reply:
The file [mixture_kalman_filter.py](dynamax/slds/mixture_kalman_filter.py)

Reply:
Note that the file [dynamax/slds/mixture_kalman_filter.py](https://github.com/probml/dynamax/pull/323/files/8eba816a359b5da89362b9e5f1de41a2d6f382e9#diff-417f39db469fedb6e21fb59fb1e89a48b943e5e4d2e3903da07aa1cdcc1cdd4f) is an exact copy of the original implementation from https://github.com/probml/JSL/blob/main/jsl/lds/mixture_kalman_filter.py which implemented a limited version of the RBPF for a tracking example. It is included here for the purpose of testing. The general algorithm where all parameters depend on the discrete latents is implemented in the function `_conditional_kalman_step`  of dynamax/slds/inference.py.

Reply:
Oh, I see. Maybe rename the file to 'mixture_kalman_filter_demo.py' or something, to make it clear that this is not the general library?